src/Controller/SantaController.php

```diff
@@ -155,57 +155,49 @@ public function message(Rudolph $rudolph, FormFactoryInterface $formFactory, Req
         $form->handleRequest($request);
 
         if ($form->isSubmitted()) {
-            $message = $form->getData()['message'] ?? '';
-            $session->set('message', $message);
-            foreach ($form->getData() as $data => $note) {
-                if (str_contains($data, 'notes-')) {
-                    $notes[str_replace('notes-', '', $data)] = $note;
-                }
-            }
-            $session->set('notes', $notes);
+            $session->set('config', $config);
+
+            $errors = array_map(function (FormError $error) {
```

Comment:
could you move the errors handling after the block `if ($form->isValid()) {`? We do not need it if the form is valid :wink:

Reply:
You're right, fixed 😊